Jenkinsfile

```diff
@@ -259,9 +261,13 @@ def withBeatsEnv(Map args = [:], Closure body) {
           cmd(label: 'Download modules to local cache - retry', script: 'go mod download', returnStatus: true)
         }
         body()
+      } catch(err) {
+        // Upload the generated files ONLY if the step failed. This will avoid any overhead with Google Storage
+        upload = true
+        error("Error '${err.toString()}'")
```

Comment:
Don't the single quotes here break interpolation of the variable? I would have thought this would have been printed literally as `${err.toString()}` but perhaps not.

Reply:
The whole string is in double quotes, so groovy interpolation happens before the cosmetic `'`